pyamg/classical/air.py

```diff
@@ -0,0 +1,227 @@
+"""Approximate ideal restriction AMG"""
+from __future__ import absolute_import
```

Comment:
do we need the `from __future__`  and  `__docformat__` lines?

Reply:
I don't know, @lukeolson should weigh in there

Reply:
I'm not aware of any `__future__` that we need.

Also we've been moving to relative imports (there was a time when this helped with circular imports if IIRC).  For consistency we should do that but we could consider going all absolute too.

Reply:
@lukeolson @bensworth  OK, I'll remove the absolute import line.

Do wee need the below `__docformat__ = "restructuredtext en"` line?